src/validation.h

```diff
@@ -305,7 +305,7 @@ void PruneAndFlush();
 
 /** (try to) add transaction to memory pool **/
 bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
-                        bool* pfMissingInputs, bool fOverrideMempoolLimit=false, bool fRejectAbsurdFee=false, bool fDryRun=false);
+                        bool* pfMissingInputs, bool fOverrideMempoolLimit=false, const CAmount nAbsurdFee=0, bool fDryRun=false);
 
```

Comment:
I think Bitcoin removes the default value for nAbsurdFee

Reply:
Develop branch of Dash.

Reply:
Bitcoin added it with the default = 0 and later removed the default:
https://github.com/bitcoin/bitcoin/blob/master/src/validation.h#L310

We can remove the default later.  I traced the code and it shouldn't cause any harm.